db/migrate/20220909220449_add_webpush_app.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "webpush"
+
+class AddWebpushApp < ActiveRecord::Migration[6.1]
+  def up
+    vapid_keypair = Webpush.generate_key.to_hash
+    app = Rpush::Webpush::App.new
+    app.name = "webpush"
+    app.certificate = vapid_keypair.merge(subject: APP_CONFIG["contact_email"]).to_json
```

Comment:
please make it fail if it's unset

```suggestion
    app.certificate = vapid_keypair.merge(subject: APP_CONFIG.fetch("contact_email")).to_json
```